--- debian/copyright	2013-08-11 13:27:30.000000000 +0000
+++ debian/copyright	2013-08-11 14:16:47.000000000 +0000
@@ -36,6 +36,11 @@
  USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
  DAMAGE.
 
+ Note: a copy of available userland source is included in the source
+ package, however some tools don't have source available and the
+ version of the source may or may not match the version of the
+ binaries.
+
 Files:
   opt/vc/include/*
 Copyright:
@@ -43,22 +48,3 @@
  2006-2012 Broadcom Europe Ltd.
 License: MIT or Apache-2 (FIXME)
 
-Files:
- boot/kernel*.img
-Copyright: 1991-2012 Linus Torvalds and many others
-License: GPL-2
- This package is free software; you can redistribute it and/or modify
- it under the terms of the GNU General Public License version 2 as
- published by the Free Software Foundation.
- .
- This package is distributed in the hope that it will be useful,
- but WITHOUT ANY WARRANTY; without even the implied warranty of
- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- GNU General Public License for more details.
- .
- You should have received a copy of the GNU General Public License
- along with this package; if not, write to the Free Software
- Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
- .
- On Debian systems, the complete text of the GNU General Public License version
- 2 can be found in `/usr/share/common-licenses/GPL-2'.
